lib/actions/app.coffee

```diff
@@ -57,7 +57,7 @@ exports.create =
 				resin.models.application.create(params.name, options.type).nodeify(callback)
 
 			(applicationId, callback) ->
-				console.info("Application created: #{params.name} (#{options.type}, id #{applicationId})")
+				console.info("Application created: #{params.name} (#{options.type}, id #{applicationId.id})")
```

Comment:
Please rename `applicationId` to just `application`, since `applicationId.id` looks very weird.